### PR TITLE
Instantiate sequalize and connect to the database

### PR DESCRIPTION
**Pull request by Saatwik-S**
This pull request consists of:
- Creation of Patient Table with few basic columns
- Instantiation of Sequalize
- Addition of dotenv module

In order to connect to the database, one should make a .env file in the main directory ( the directory that contains index.js ) and then add the following variables:

- **DBNAME**
- **DBPASSWORD**
- **DBUSER**
- **DBHOST**
- **DBPORT**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,15 +6,18 @@
   "scripts": {
     "test": "nodemon src/index.js"
   },
-  "author": "Mckinsey and Company",
+  "author": "McKinsey and Company",
```
Comment (rkdixit96) [inline package.json:9]: Remove this author

File: src/database/models/patientModel.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		type: DataTypes.STRING,
+	},
+	mobileNumber: {
+		type: DataTypes.STRING,
+	},
+});
+
+Patient.sync({force: true}).then(console.log('Successfully Synchornized the Database with Patient table')).catch(console.log);
+
+module.exports = Patient;
```
Comment (rkdixit96) [inline src/database/models/patientModel.js:25]: Eslint is not working or is not configured properly

File: src/routes/onboarding/onboardingRoutes.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const { handleNewDoctorOnboardingRequest } = require('../../functions/onboarding/doctor/createDoctor');
+
+const router = require('express').Router();
+
+
+router.route('/doctor')
+	.post(handleNewDoctorOnboardingRequest);
+
+module.exports = router;
```
Comment (rkdixit96) [inline src/routes/onboarding/onboardingRoutes.js:9]: General rule - If the folder is called route - no need to mention again in file name. Please change even the model file as well

File: src/database/models/patientModel.js (hunk 2 above)
Comment (rkdixit96) [reply src/database/models/patientModel.js:25]: Lets use predefined style for linting

**Review by rkdixit96 — CHANGES_REQUESTED** [review]

**Review by rkdixit96 — APPROVED** [review]

File: src/index.js [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
 
+app.use('/', (request, response) => {
+	response.send('Welcome to the backend API');
 });
 
+app.use('/onboarding', onboardingRoutes);
+
+
```
Comment (NiranjanBorawar) [inline src/index.js:21]: Let's configure prettier also
or any other default file formatter

Comment (NiranjanBorawar) [reply src/index.js:21]: It will avoid these whitespaces and newlines

**Pull request closed without merging**